ansible/configure-jvb-local-oracle.yml

```
@@ -208,7 +208,7 @@
         telegraf_inputs:
           prometheus:
             urls: ["http://localhost:8080/metrics"]
-            namepass: "{{ if jvb_enable_telegraf }}{{ telegraf_jvb_filter }}{{ else ['jitsi_jvb_healthy', 'jitsi_jvb_loss_*'] }}"
+            namepass: "{{ if jvb_enable_telegraf }}{{ telegraf_jvb_filter }}{{ else ['jitsi_jvb_healthy', 'jitsi_jvb_loss_*'] }}{{ endif }}"
```

I think this may work and is easier to read
```suggestion
            namepass: "{{ telegraf_jvb_filter if jvb_enable_telegraf else ['jitsi_jvb_healthy', 'jitsi_jvb_loss_*'] }}"
```